src/model/ScheduleTypeRegistry.cpp

```diff
@@ -217,6 +217,7 @@ namespace model {
        OptionalDouble()},
       {"CoilHeatingWater", "Availability", "availabilitySchedule", false, "Availability", 0.0, 1.0},
       {"CoilHeatingWaterBaseboard", "Availability", "availabilitySchedule", false, "Availability", 0.0, 1.0},
+      {"CoilWaterHeatingAirToWaterHeatPumpVariableSpeed", "Grid Signal", "gridSignalSchedule", true, "", OptionalDouble(), OptionalDouble()},
```

Comment:
@joseph-robertson What is this? I can't find it anywhere in the IDD nor as a method.

Reply:
Removed in 1ca23846362495dc84903eb821a16b0b645baec6